RecoLocalMuon/GEMSegment/plugins/SealModule.cc

```diff
@@ -4,6 +4,15 @@
 #include <RecoLocalMuon/GEMSegment/plugins/ME0SegmentProducer.h>
 #include <RecoLocalMuon/GEMSegment/plugins/ME0SegmentBuilderPluginFactory.h>
 #include <RecoLocalMuon/GEMSegment/plugins/ME0SegAlgoMM.h>
+#include <RecoLocalMuon/GEMSegment/plugins/ME0SegAlgo.h>
 
 DEFINE_FWK_MODULE(ME0SegmentProducer);
 DEFINE_EDM_PLUGIN(ME0SegmentBuilderPluginFactory, ME0SegAlgoMM, "ME0SegAlgoMM");
+
+#include <RecoLocalMuon/GEMSegment/plugins/GEMSegmentProducer.h>
+#include <RecoLocalMuon/GEMSegment/plugins/GEMSegmentBuilderPluginFactory.h>
+#include <RecoLocalMuon/GEMSegment/plugins/GEMSegAlgo.h>
+
+DEFINE_FWK_MODULE(GEMSegmentProducer);
```

Comment:
``` DEFINE_FWK_MODULE ``` can be done in the GEMSegmentProducer.cc, which is a more common way since ~2010